husky_components/datagrid/main.js

```diff
@@ -2311,6 +2324,11 @@
                             searchFields: this.options.searchFields.join(',')
                         });
                     }
+                    if (searchString !== '') {
+                        this.storage.set('url', url);
+                    } else {
+                        this.storage.remove('url');
```

Comment:
Is that the correct behavior? I think this also makes us lose the sorting and that kind of stuff, only because the search field was emptied.